resources/zib/zib-HealthcareProvider-Organization.xml

```diff
@@ -87,7 +87,7 @@
       <sliceName value="departmentSpecialty" />
       <short value="DepartmentSpecialty" />
       <definition value="The specialty of the healthcare provider’s department. The departmental specialty can be filled in if further indication of the healthcare provider is needed. This refers to the recognized medical specialties as stated in the BIG Act." />
-      <comment value="The ValueSet binding strenght deviates from the zib because the binding strenght must be set to required in order to make the discriminator work. However, the open slice definition allows to extend codes." />

```

Comment:
Je kan je afvragen of dit commentaar uberhaupt wel zinvol is. We voeren dit in ieder geval niet consistent door.

Reply:
Mee eens! Zal ik deze comment dan gewoon weghalen?

Reply:
Ja, dat lijkt me beter.

Reply:
Comment is verwijderd.